net/nettest/conn.go

```diff
@@ -5,21 +5,13 @@
 package nettest
 
 import (
-	"io"
+	"net"
 	"time"
 )
 
 // Conn is a bi-directional in-memory stream that looks like a TCP net.Conn.
 type Conn interface {
```

Comment:
aside: this is a very useful type and not entirely related to testing (at least how we plan to use it), but I guess it works here for now.